SMBPositionAnalytics.md

```diff
@@ -0,0 +1,136 @@
+SMB Position Analytics Service
+==============================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Carotene With Location Request](#carotene-with-location-request)
+- [Carotene Response Structure](#carotene-response-structure)
+- [Job Request](#job-request)
+- [Job Response Structure](#job-response)
+- [Versioning](#versioning)
+
+## Summary
+
+The Small Medium Business(SMB) Position Analytics service provides hiring data pertaining to job titles and currently open positions available on CareeBuilder job boards. The service has two endpoints:
+
+https://api.careerbuilder.com/core/positionanalytics/carotene
+
+https://api.careerbuilder.com/core/positionanalytics/job
+
+Requests can be sent in the form of HTTP GET or POST (form or JSON)
+
+The data returned will contain an object or group of objects representing actions taken on a carotene title or job respectively. Each action will have associated `current` and `delta` values. The `current` value will account the total number an action was performed over the last day; the `delta` will identify the change in percentage of the `current` value from the day before. Positive values in the `delta` identify an increase day over day and negative values identify a decrease.
+
+## Carotene With Location Request
+A request to the carotene endpoint provides data on how a specified job title performs in a given location.
+
+E.g. I'm a recruiter and I'm about to post a Software Engineer job in Atlanta. How many applies have their been to Software Engineer titles in this city in the last day? What is the change from the day before? On average how long does it take to fill a Software Engineer role in this location? This endpoint serves to answer these questions.
+
+Requests consist of:
+
+| Param | Type | Description |
+|-------|------|-------------|
+| `carotene_id` | string | Unique carotene identifier v3.1.
+| `city` | string | Full name of a city respective of the carotene title
+| `state` | string | Two letter abbreviation for the state
+
+Example cURL request:
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/positionanalytics/carotene \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"carotene_id": "11.0",
+	"city": "Atlanta",
+	"state": "GA",
+      }'
+```
+
+Currently only Carotene v3.1 is supported. See the documentation for [Job Title service](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/JobTitle.md)
+for more information about carotene classification and carotene IDs.
+
+## Carotene Response Structure
+
+ `num_of_applies` holds the `current` number of applies within the last day period and `delta` percent change in applies from the previous day.
+
+ `avg_time_to_fill` is the time period in days on average to fill a position identified by the `carotene_id`.
+
+ `timestamp` is the time in which the data was last refreshed.
```

Comment:
May be a good idea to clarify the format of the timestamp. It does appear to be _yyyymmddhhmmss_, right?

Reply:
Also, and I'm not sure this is a good idea but wanted to ask anyway, perhaps also mention how often we intend to refresh the data? Not the exact frequency we'll strive to maintain, but something that can give the client an idea whether we'll promise to update daily, weekly or several times a day?